lib/improver/lapse_rate.py

```diff
@@ -460,8 +460,7 @@ def process(self, temperature_cube, orography_cube, land_sea_mask_cube):
             lapse_rate_cube_list.append(lapse_rate_slice)
 
         lapse_rate_cube = lapse_rate_cube_list.merge_cube()
-        lapse_rate_cube.rename('temperature_lapse_rate')
-        lapse_rate_cube.long_name = "lapse_rate"
+        lapse_rate_cube.rename('lapse_rate')
```

Comment:
I think the correct CF standard name we want here is "air_temperature_lapse_rate".

Reply:
I've corrected to use "air_temperature_lapse_rate".